src/vs/workbench/contrib/chat/browser/attachments/implicitContextAttachment.ts

```diff
@@ -62,20 +63,27 @@ export class ImplicitContextAttachmentWidget extends Disposable {
 		const friendlyName = `${fileBasename} ${fileDirname}`;
 		const ariaLabel = range ? localize('chat.fileAttachmentWithRange', "Attached file, {0}, line {1} to line {2}", friendlyName, range.startLineNumber, range.endLineNumber) : localize('chat.fileAttachment', "Attached file, {0}", friendlyName);
 
-		const uriLabel = this.labelService.getUriLabel(file, { relative: true });
 		const currentFile = localize('openEditor', "Current file context");
 		const inactive = localize('enableHint', "disabled");
-		const currentFileHint = currentFile + (this.attachment.enabled ? '' : ` (${inactive})`);
-		const title = `${currentFileHint}\n${uriLabel}`;
+		const currentFileHint = new MarkdownString(currentFile + (this.attachment.enabled ? '' : ` (${inactive})`));
+
+		const title = [currentFileHint, ...this.getUriLabel(file)]
```

Comment:
Right, I've made the `getUriLabel` to return a list of markdown strings for it to be consistent with other similar functions where we return the label, and because of that, to make the array type consistent, I had to use a markdown string here. Can keep the `currentFileHint` part as a string and add it afterwards if that bothers you 🤗

Reply:
> with other similar functions where we return the label

Which functions? If you aren't intending the text to be rendered as markdown, I'd say there's no need to pass it around as MarkdownStrings

Reply:
Like [ChatDynamicVariables.getHoverForReference()](https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/chat/browser/contrib/chatDynamicVariables.ts#L111C56-L111C70)(or [decoration options](https://github.com/microsoft/vscode/blob/f408139c28c2c653d673e9a85ba250d6c6a588be/src/vs/editor/common/editorCommon.ts#L732))